src/decisionengine/framework/dataspace/dataspace.py

```diff
@@ -9,6 +9,8 @@
     "DataSpace",
 ]
 
+logger = structlog.getLogger("decision_engine")
+logger = logger.bind(module=__name__.split(".")[-1])
```

Comment:
I'm not sure I understand why this module whet from `self.logger` to just `logger` when it appears most of the other modules swapped out their internal `self.logger` for a `structlog` based `self.logger`

Reply:
The 'logger' is a structlog here, but I have now given the class a self.logger for consistency. Thanks!